### PR TITLE
Capstone Project 1

### PR DESCRIPTION
**Pull request by cnualartc**
In this project: 

- Created a home page and an about page following the design guidelines.
- Personalized the design to be the website of a fictitious rock fest.
- Created JS code to make the "more" button work correctly.
- Website is responsive to two screen sizes.
- Menu on the mobile version opens and closes with the "hamburger" button (I tried adding the X as a close button but I figured it looks a lot better to use the same "hamburger" menu to open and close (toggle)).

Lint to video description: https://www.loom.com/share/8fc6adf91ba74a2eb8602840b8fa6b53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,126 @@
...
+
+
+### Key Features <a name="key-features"></a>
+
+- **Web page with previews of my projects**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove the empty space

File: README.md [hunk 2]
```diff
@@ -1,2 +1,126 @@
...
+## 💻 Getting Started <a name="getting-started"></a>
+
+The user will be able to see the projects I have created and collaborated in through-out my career as a web developer.
+
+To get a local copy up and running, follow these steps.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+You can view this project live using: https://github.com/cnualartc/Capstone_project_1
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly put the correct demo link here. The current one is not working

File: README.md [hunk 3]
```diff
@@ -1,2 +1,126 @@
...
+Special thanks to Cindy Shin, creator of the original design.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
```
Comment (Kossi-stack) [inline README.md]: - [x] Kindly remove the empty spaces.

File: home.html [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+  </section>
+  <footer>
+    <div class="container">
+      <div class="row row-cols-md-2">
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+          <img class="positive" src="./images/footer/positive.png" alt="creative commons korea logo in positive">
+        </div>
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+            <p>2023 Creative Commons korea.<br>Some Rights Reserved.</p>
+        </div>
```
Comment (Kossi-stack) [inline home.html]: - [x] Kindly fix the space between the image and the copyright.

Screenshot 2023-02-17 at 15 06 53

File: about.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+  <footer>
+    <div class="container">
+      <div class="row row-cols-md-2">
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+          <img class="positive" src="./images/footer/positive.png" alt="creative commons korea logo in positive">
+          <img class="negative" src="./images/footer/negative.png" alt="creative commons korea logo in negative">
+        </div>
+        <div class="col-6 d-flex justify-content-center cc-logo-container">
+            <p>2023 Creative Commons korea.<br>Some Rights Reserved.</p>
+        </div>
```
Comment (Kossi-stack) [inline about.html]: - [x] Kindly fix the space between the image and the copyright.

Screenshot 2023-02-17 at 15 03 25

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @cnualartc,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️ 

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: about.html (hunk 5 above)
Comment (cnualartc) [reply about.html]: Thank you so much for the review @Kossi-stack . I made all the changes mentioned.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,111 @@
 # Capstone_project_1
 First Microverse capstone project (module 1-block 5)
+
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="./images/my_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (Lucas-Masaba) [inline README.md]: - [x] Your image is not displaying on the README. You can add your own logo or if you don't have one, leave the README without a logo.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,111 @@
...
+## 💻 Getting Started <a name="getting-started"></a>
+
+The user will be able to see the projects I have created and collaborated in through-out my career as a web developer.
+
+To get a local copy up and running, follow these steps.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+You can view this project live using: https://cnualartc.github.io/Capstone_project_1/
```
Comment (Lucas-Masaba) [inline README.md:55]: - [x] Your live demo link redirects to a page like that in the image below. Please make sure it redirects to the website you created
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @cnualartc  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. You are almost there.
almost-there

 ## Highlights 🍾 
- Made changes as requested by the previous reviewer 👍 
- Linter checks passing ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello @cnualartc  , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Proffessional README








 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**